packages/vite/package.json

```diff
@@ -31,7 +31,7 @@
   "dependencies": {
     "@phenomnomnominal/tsquery": "~5.0.1",
     "@swc/helpers": "~0.5.0",
-    "dotenv": "~16.3.1",
```

Comment:
I thought we still need dotenv

Reply:
create a separate pr for this, no loner needed in the package.json